### ISSUE TITLE
Can't have multiple vue-draggable components on same page

### ISSUE THREAD

**Issue by sgarner** [Vivify-Ideas/vue-draggable#27]
**Describe the bug**
When multiple components on the same page contain `v-drag-and-drop` directives, events are fired in the context of the last such component on the page rather than the one where the interaction occurs.

**To Reproduce**
Following codesandbox (forked from one of the examples) demonstrates this behavior: https://codesandbox.io/s/o4vv7qz52z

In this fork, the main `App.vue` contains two components:

```html
  <VueDraggable id="vd1"/>
  <VueDraggable id="vd2"/>
```

Inside `VueDraggable.vue`, an `onDrop` event handler is registered like this:

```js
  options: {
    reactivityEnabled: true,
    onDrop: event => {
      console.log("onDrop fired", event, this.$el);
    }
  }
```

**Expected behavior**
When dragging items within the first (top) component, it is expected that `this.$el` in the console log references `<div id="vd1">`, and in the second (bottom) component it should reference `<div id="vd2">`.

However, in fact `this` in the event handler always references the second component (with `<div id="vd2">`).

![image](https://user-images.githubusercontent.com/904004/55601977-ec84b180-57bf-11e9-8e83-a5356eca16df.png)

**Additional context**

I'm not sure if this is related to #12. The symptoms are different - `this` does refer to the Vue component, just the wrong one. And the solution linked in that issue doesn't help.

**Comment by nikolasp** [comment]
Hi @sgarner,

Thanks for the feedback. 
I'll check this and let you know if anything changes.

**Comment by nikolasp** [comment]
Hi @sgarner,

I've published new version 2.0.1 which should fix the issue. 
Tested on your CodeSandbox example and everything works as expected.
Please, check updated version and let me know if you have further issues. 

Thanks again for the feedback. :)

**Comment by aislanmaia** [comment]
I have a similar problem! 
In the [codesandbox](https://codesandbox.io/s/vue-draggable-6igf3) that I changed a little bit to have two instance components (and the vue-draggable is in the 2.0.0 version), and the drag and drop between the elements from the instances work as expected. 

But in my Vue project, that the `vue-draggable` is in 2.0.3 version, the drag between the two instances doesn't work. 
In the `onDragend` sometimes the droptarget is null, sometimes the droptarget is the same origin where I pick the element, which is wrong.
